### DISCUSSION TITLE
Tile error when trying to zoom in on WMS layer

### DISCUSSION THREAD

**Discussion by lstillwe** [TerriaJS/terriajs#6527] [Q&A]
Hello! I am trying to us a some WMS services provided by NowCoast/NOAA.
They are described here: https://nowcoast.noaa.gov/arcgis/rest/services/nowcoast/wwa_meteocean_tropicalcyclones_trackintensityfcsts_time/MapServer/

And here is my catalog entry for one of these layers:
{
           "id": "HL1_DEMO"
            "name": "Hurricane HL1 - DEMO",
            "type": "group",
            "members": [
                {
                    "id": "HL1-9",
                    "url": "https://nowcoast.noaa.gov/arcgis/services/nowcoast/wwa_meteocean_tropicalcyclones_trackintensityfcsts_time/MapServer/WMSServer?request=GetCapab
ilities&service=WMS",
                    "name": "Tropical Cyclone Center Position Forecasts",
                    "show": true,
                    "type": "wms",
                    "layers": "9",
                    "description": "For more information see: https://nowcoast.noaa.gov/arcgis/rest/services/nowcoast/wwa_meteocean_tropicalcyclones_trackintensityfcsts_ti
me/MapServer/",
                    "dataCustodian": "NOAA"
                }
}

The layer displays correctly, when I first add to the map, however when I try to zoom in/out a couple of times, I get this error:
![Screen Shot 2022-09-02 at 12 07 24 PM](https://user-images.githubusercontent.com/1794116/188194770-e76a5136-bf1b-457a-8234-8082cdbc5bde.png)

Does anyone know what might be causing this issue? Thanks very much!

**Comment by ppasq** [comment]
Hi! I'm experiencing the same behaviour with GeoServer layers via WMS but only when I switch to the 2D Map View. When I set the map to 3D the WMS works fine.

**Reply by lstillwe** [reply]
Hello - Yes, in this case we were being throttled by the NOAA server - too many requests within a certain time period. We had to abandon the use of those datasets.
For your case try running TerriaMap in Chrome. I know for sure 2D does not work in Safari.

**Reply by ppasq** [reply]
Yes, it is a Safari issue only. Thanks for your reply.
